src/net/dcsctp/packet/crc32c.cc

```diff
@@ -11,7 +11,7 @@
 
 #include <cstdint>
 
-#include "third_party/crc32c/src/include/crc32c/crc32c.h"
+#include "crc32c/crc32c.h"
```

Comment:
Shouldn't it use braces? It's not an in-project path anymore.

Reply:
sure